test/boost/schema_change_test.cc

```diff
@@ -779,6 +779,7 @@ future<> test_schema_digest_does_not_change_with_disabled_features(sstring data_
     }
     cql_test_config cfg_in(db_cfg_ptr);
```

Comment:
> In the following commit, we remove the `consistent-topology-changes`
> experimental feature.

Nit: IIUC we don't really remove it in the next commit, we only remove any effect it might have on the code.

Reply:
I fixed it in this commit and some other commits.